src/fairchem/core/trainers/ocp_trainer.py

```diff
@@ -430,7 +430,9 @@ def predict(
             desc=f"device {rank}",
             disable=disable_tqdm,
         ):
-            with torch.cuda.amp.autocast(enabled=self.scaler is not None):
+            with torch.cuda.amp.autocast(
+                enabled=self.scaler is not None and not self.cpu
```

Comment:
enabled=self.scaler is not None 
should be fine? 
since self.scaler is None if self.cpu is True?

Reply:
ah good catch, I made this change first then realized the other one after.